### PR TITLE
Candidate and Non-candidate PAC pages

### PR DESCRIPTION
**Pull request by alexisbarnes**
- Candidate PAC links to candidate page (per Jeremy's request)
- Non-candidate PACs have their own page:
Screen Shot 2021-04-26 at 8 01 45 PM

**KNOWN BUGS**

- I made the NULLs return on the search results page 😅:
Screen Shot 2021-04-26 at 8 02 07 PM

- Links on the Contributors page are broken if the contributor has given to a PAC. The PAC shows as "NULL NULL NULL":
Screen Shot 2021-04-26 at 8 06 42 PM

- Download Results not working (error: "unable to process request")
Screen Shot 2021-04-26 at 8 12 18 PM

- The test build fails (getting this via email from Github)

Made lots to changes so open to feedback or requests. Thanks!! 😊

### CODE REVIEW

File: server/lib/queries.js [hunk 1]
```diff
@@ -52,6 +52,47 @@ const getCandidateSummary = async (ncsbeID) => {
...
+const getCommitteeSummary = async (ncsbeID) => {
+  // Post MVP we should probably find a way to speed this up.
+  console.time('getCommitteeSummary')
+  const summary = await db.query(
+    `
+    with aggregated_contributions as (
+      select count(*)    as aggregated_contributions_count,
+             sum(amount) as aggregated_contributions_sum
+      from contributions
+      where contributor_id IS NULl
```
Comment (SchaeStewart) [inline server/lib/queries.js:77]: typo: change `NUL1` to `NULL`

**Review by SchaeStewart — CHANGES_REQUESTED** [review]
@alexisbarnes it looks pretty good overall. Congrats on making your first backend PR (I think)!

I do think we should add additional tests and add to the `api_doc.md` before the changes are merged though.  
For the tests you should be able to copy what's already there for the candidates.

Regarding the NULLs coming back, when was the last time you ran `make clean-db`? If you did it before my fix for that was merged it could explain why they returned.

Regarding the failing CSV download, you should be able to check the logs and see the error. If you need additional help, just let me know. I'm happy to pull down the changes and take a look :smile: 

Great job overall, I hope the API wasn't too confusing.

**Comment by alexisbarnes** [comment]
> @alexisbarnes it looks pretty good overall. Congrats on making your first backend PR (I think)!
> 
> I do think we should add additional tests and add to the `api_doc.md` before the changes are merged though.
> For the tests you should be able to copy what's already there for the candidates.
> 
> Regarding the NULLs coming back, when was the last time you ran `make clean-db`? If you did it before my fix for that was merged it could explain why they returned.
> 
> Regarding the failing CSV download, you should be able to check the logs and see the error. If you need additional help, just let me know. I'm happy to pull down the changes and take a look 😄
> 
> Great job overall, I hope the API wasn't too confusing.

It is my first backend PR! 😭

I haven't ran `make clean-db` in a while so I will try that. The API is well organized and straightforward to follow! 🎉  Thanks for the feedback!

**Comment by mcelestini** [comment]
@alexisbarnes this is awesome 👏  I just test drove it and it works like a charm. I think we can fix the individual contributor page not showing committee name and going to the candidate page instead of the committee page with the following diff. This is kind of hard to read in a comment so let me know if you've got any questions about this, but if you make this change to `useTableColumns` at line `132`, this should do the trick with showing the committee name and going to the right page

```-        accessor: ({ candidate_full_name, committee_sboe_id }) => (
-          <Link
-            to={(location) => ({
-              pathname: CANDIDATE_URL + committee_sboe_id,
-              fromPathname: location.pathname,
-            })}
-          >
-            {' '}
-            {candidate_full_name}
-          </Link>
-        ),
+        accessor: ({
+          candidate_full_name,
+          committee_sboe_id,
+          committee_name,
+        }) =>
+          Boolean(candidate_full_name) ? (
+            <Link
+              to={(location) => ({
+                pathname: CANDIDATE_URL + committee_sboe_id,
+                fromPathname: location.pathname,
+              })}
+            >
+              {' '}
+              {candidate_full_name}
+            </Link>
+          ) : (
+            <Link
+              to={(location) => ({
+                pathname: COMMITTEE_URL + committee_sboe_id,
+                fromPathname: location.pathname,
+              })}
+            >
+              {' '}
+              {committee_name}
+            </Link>
+          ),```

File: src/components/Committee.js [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+                value={summary.max}
+                displayType="text"
+                decimalScale={2}
+                fixedDecimalScale={true}
+                thousandSeparator={true}
+                prefix="$"
+              />
+            </p>
+            <p className="summary-stat">
+              Total Number of Aggregated Contributions:
```
Comment (mcelestini) [inline src/components/Committee.js:151]: I don't remember if we have this data yet, do we want to keep it in here?

**Review by mcelestini — APPROVED** [review]

**Comment by alexisbarnes** [comment]
@SchaeStewart I think I fixed all the issues. How does it look?

**Review by SchaeStewart — APPROVED** [review]
Looks good! Merge whenever you're ready

**Pull request merged**